crates/types/src/data.rs

```diff
@@ -845,3 +847,46 @@ pub mod null_block {
         }
     }
 }
+
+/// A packed bundle constructed from a sequence of bundles.
+#[derive(Debug, Eq, PartialEq, Clone)]
+pub struct PackedBundle<TYPES: NodeType> {
+    /// The combined transactions as bytes.
+    pub encoded_transactions: Arc<[u8]>,
+
+    /// The metadata of the block.
+    pub metadata: <TYPES::BlockPayload as BlockPayload<TYPES>>::Metadata,
+
+    /// The view number that this block is associated with.
+    pub view_number: TYPES::Time,
+
+    /// The bid fees for submitting the block.
+    pub bid_fees: Vec1<BuilderFee<TYPES>>,
```

Comment:
We were using Vec1 to make this backwards-compatible right?   But old blocks won't have a bid fee at all.  Does that cause problems here?